public/api-merged.json

```diff
@@ -8524,7 +8524,7 @@
         "tags": [
           "service_accounts"
         ],
-        "summary": "Create service account",
+        "summary": "# Create service account",
```

Comment:
@papagian Are these `#` at the front of these correct? 

It appears most `summary` fields in the spec don't have them?

Reply:
latest `gofmt` adds them [if there is a missing a trailing dot](https://github.com/grafana/grafana/blob/main/pkg/services/serviceaccounts/api/api.go#L86).

I doesn't harm that much; I can fix them later